src/Cli/OrgCommands.php

```diff
@@ -251,6 +255,71 @@ public function orgUpdateProjectsInfo($csv_file, $options = [
         }
     }
 
+    /**
+     * @command org:update-projects-merge-prs
+     * @param $csv_file The path to csv file that contains projects information.
+     */
+    public function orgUpdateProjectsMergePrs($csv_file, $options = [
+        'as' => 'default',
+        'age' => 30,
+        'pr-title-query' => '[UpdateTool - Project Information]',
+        'pr-title-pattern' => '',
+    ])
+    {
+        $api = $this->api($options['as']);
+        $prTitleQuery = $options['pr-title-query'];
+        $prTitlePattern = $options['pr-title-pattern'];
+        $age = $options['age'];
+
+        if (file_exists($csv_file)) {
+            $csv = new \SplFileObject($csv_file);
+            $csv->setFlags(\SplFileObject::READ_CSV);
+            foreach ($csv as $row_id => $row) {
+                // Skip header row.
+                if ($row_id == 0) {
+                    continue;
+                }
+                if (empty($row[0])) {
+                    // Empty line, probably EOF. Break loop.
+                    break;
+                }
+                $projectName = $row[2];
+                $projectFullName = $row[3];
+                $projectOrg = $row[5];
+                $projectSupportLevel = $row[23];
```

Comment:
If we can't search across the whole org & find PRs in any project, then we should find the column numbers by label, as is done in the other csv file command.